packages/ag-charts-enterprise/src/features/error-bar/errorBar.ts

```diff
@@ -4,23 +4,34 @@ import type { ErrorBarPoints, ErrorBarNodeProperties } from './errorBarNode';
 import { ErrorBarNode } from './errorBarNode';
 import { ERROR_BARS_THEME } from './errorBarTheme';
 
-const { mergeDefaults, ChartAxisDirection, Validate, BOOLEAN, OPT_STRING, NUMBER } = _ModuleSupport;
+const {
+    mergeDefaults,
+    valueProperty,
+    ChartAxisDirection,
+    Validate,
+    NUMBER,
+    STRING,
+    OPT_BOOLEAN,
+    OPT_COLOR_STRING,
+    OPT_NUMBER,
+    OPT_STRING,
+} = _ModuleSupport;
 const { Logger } = _Util;
-type AnyScale = _Scale.Scale<any, any, any>;
 
+type AnyScale = _Scale.Scale<any, any, any>;
 type OptionalErrorBarNodeProperties = { [K in keyof ErrorBarNodeProperties]?: ErrorBarNodeProperties[K] };
 
 class ErrorBarCapConfig implements OptionalErrorBarNodeProperties {
-    @Validate(BOOLEAN)
-    visible?: boolean = undefined;
+    @Validate(OPT_BOOLEAN)
+    visible?: boolean;
```

Comment:
Q: Does this work without the `= undefined` - i.e. can a user specify `visible` without an options binding error?

Reply:
Oddly enough yes (https://plnkr.co/edit/m2TZ2vDoTYoHO6ra?open=main.js), although I'm unsure why. IIRC from another conversation that we had, this property shouldn't exist unless its set to something 🤷🏻‍♂️

Reply:
I've rebased & amended the commit that removes the explicit `= undefined` just to be safe.